setup/configure_tpm_emulator/swtpm_permissive.te

```diff
@@ -13,3 +15,9 @@ dontaudit swtpm_t self:tcp_socket { accept listen };
 dontaudit swtpm_t tcsd_var_lib_t:dir { add_name remove_name write };
 dontaudit swtpm_t tcsd_var_lib_t:file { create getattr lock open read rename setattr unlink write };
 dontaudit swtpm_t unreserved_port_t:tcp_socket name_bind;
+dontadutit swtpm_t var_lib_t:dir { add_name remove_name write };
```

Comment:
typo here? `dontadutit` also in the last 2 ones

Reply:
Good catch! Thank you!